### PR TITLE
Fix small typos

### PR DESCRIPTION
**Pull request by sindelio**

### CODE REVIEW

**Review by Trott — APPROVED** [review]

File: locale/en/about/privacy.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Unless specifically indicated within a Site, the Sites are not intended for mino
...
 
-In order to tailor the Foundation’s subsequent communications to users and continuously improve the Sites’ products and services, the Foundation may also ask users to provide information regarding their interests, demographics, experience and detailed contact preferences. the Foundation and third party advertising companies may track information concerning a user’s use of the Sites, such as a user’s IP address.
+In order to tailor the Foundation’s subsequent communications to users and continuously improve the Sites’ products and services, the Foundation may also ask users to provide information regarding their interests, demographics, experience and detailed contact preferences. The Foundation and third party advertising companies may track information concerning a user’s use of the Sites, such as an user’s IP address.
```
Comment (richardlau) [inline locale/en/about/privacy.md:23]: `a user` is correct; `an user` is wrong (at least in British English).

**Review by willin — APPROVED** [review]
see richardlau's comment

**Comment by stevemao** [comment]
@sindelio why closed? `The Foundation` part is correct

**Comment by sindelio** [comment]
I made another pull request with just the correct part, and closed this one

**Pull request closed without merging**